.github/workflows/lin_qt.yml

```diff
@@ -0,0 +1,180 @@
+name: "Linux dependencies: Qt"
+
+env:
+    workflow_name: lin_qt
+    openssl_version: "1.1.1t"
+    icu_version: "72.1"
+    submodules: qtdeclarative qtimageformats qtsvg qttools qtx11extras
+    TARGET_ARCH: "x86_64"
+
+on:
+    workflow_dispatch:
+        inputs:
+          use_ccache:
+            description: 'Use ccache (for workflow debugging only!)'
+            required: false
+            type: boolean
+    schedule:
+        - cron: '30 1 * * 1' # run at 1:30 AM UTC every Monday
+    repository_dispatch:
+        types: [lin_release]
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        container: ${{ matrix.container }}
+
+        strategy:
+          fail-fast: false
+          matrix:
+            binary_compatibility:
+              - "glibc_2.23"  # Ubuntu 16.04
+              - "glibc_2.27"  # Ubuntu 18.04
+              - "glibc_2.31"  # Ubuntu 20.04, Debian 11
+            version:
+              - "5.15.9"
```

Comment:
Is it possible to pull this Qt version string to header of the action, so it's more like a declared variable?

Reply:
Actually, nevermind. I see the idea behind it now. I think we can keep it here, as is.